src/main/java/me/totalfreedom/totalfreedommod/discord/Discord.java

```diff
@@ -60,6 +62,7 @@
     public ScheduledThreadPoolExecutor RATELIMIT_EXECUTOR;
     public List<CompletableFuture<Message>> sentMessages = new ArrayList<>();
     public Boolean enabled = false;
+    private final Pattern DISCORD_MENTION_PATTERN = Pattern.compile("(<@!?([0-9]{16,20})>)");
```

Comment:
Can we add a check for `<@&ROLE-ID>` please - this pattern is for user mentions.

Reply:
it do work with role id. i just tested it on my local server

Reply:
Can you put these changes on the dev server so I can test them myself? Let me know if you can't and I'll do it.

Reply:
Just tested and I can confirm it works.